Documentation/community_features.md

```diff
@@ -57,6 +57,9 @@ Synchronization modes accessible through the "LFO SYNC" shortcut.
 ### Kit Keyboard View
  - ([#112]) All-new use for the "keyboard" button in kit clips, uses the main pad grid for MPC-style 16 level playing. Horizonatal encoder scrolls by one pad at a time, allowing positioning drums left to right, and vertical encoder jumps vertically by rows.
 
+### Quantize and Humanize Notes
+  - ([#129]) Press and hold a note in clip view and turn the tempo knob right or left to apply quantize or humanize respectively to that row. Pressing and turning the tempo knob will apply quantize or humanize to all rows. The amount of quantization/humanization is shown in the display. This feature can be toggled in the [runtime features menu](#runtime-features).
```

Comment:
You'll need to add this to the list of PRs at the end of the file for github to render the link correctly.